src/app/common/components/flows-table/flows-table.types.ts

```diff
@@ -1,10 +1,16 @@
 import { MaybeNull } from "src/app/common/app.types";
 import { DropdownSettings } from "angular2-multiselect-dropdown/lib/multiselect.interface";
-import { Account, Dataset, FlowConnectionDataFragment, FlowStatus } from "src/app/api/kamu.graphql.interface";
+import {
+    Account,
+    DatasetListFlowsDataFragment,
+    FlowConnectionDataFragment,
+    FlowStatus,
+} from "src/app/api/kamu.graphql.interface";
 
 export interface FlowsTableData {
-    connectionData: FlowConnectionDataFragment;
-    involvedDatasets: Dataset[];
+    connectionDataForTable: FlowConnectionDataFragment;
+    connectionDataForWidget: FlowConnectionDataFragment;
```

Comment:
These fragments cannot be of the same type, as the queries are different. Create a dedicated simplified fragment for tiles, and use it instead for tile data.

Reply:
Added separate type for `connectionDataForWidget`